addon/lib/metrics.js

```diff
@@ -33,6 +38,30 @@ const PREFERENCE_OVERRIDES = {
 
 let pingTimer = null;
 
+const variantMaker = {
```

Comment:
Kind of a nit: Should this stuff really be a part of `metrics.js`? We do want to send the variant info in pings, but it seems like the rest might live better in its own module. We could defer to #610 as part of other refactorings